src/Map/Map.js

```diff
@@ -145,12 +145,12 @@ class Map extends React.Component {
             fullScreen={fullScreen}
             style={style}>
             <MapLogo logoPosition={logoPosition} translations={translations} />
+            {mapInitialized // wait for map to initialize before rendering children
+              ? children
+              : null
+            }
```

Comment:
is this a breaking change? @PatchesMaps

Reply:
If an IA was expecting the children to be rendered as peers in the DOM I guess it would break. However, I believe that the original behavior could be classified as a bug since I don't see a plausible usecase to justify this behavior's existance

Reply:
good enough for me! a 🐛 it was

Reply:
make sure this does not break single maps